File: App/screens/components/Task.js
import React from 'react';
import {Text, View, StyleSheet} from 'react-native';

export default ({task = {}}) => {
  return (
    <View style={styles.listColumn}>
      <Text style={styles.listCell}>{task.Title}</Text>
      <Text style={[styles.listCell, {flex: 3}]}>{task.Description}</Text>
      <Text style={styles.listCell}>{task.Deadline}</Text>
      <Text style={styles.listCell}>{task.Status}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  listColumn: {
    flex: 1,
    flexDirection: 'row',
    textAlign: 'center',
    alignItems: 'stretch',
    marginTop: 10,
  },
  listCell: {
    flex: 1,
    textAlign: 'center',
    alignItems: 'center',
  },
});

File: AppAndroid/app/src/main/java/app/app/Model/Task.kt
package app.app.Model

data class Task(
    val ID: String,
    var Title: String,
    var Description: String,
    var Deadline: String,
    var Status: String
){
    override fun toString(): String = "$Title $Description $Deadline $Status"
    companion object{
        fun getHeader() : String {
            return "Title           Description         Deadline  Status"
        }
    }


}